custom/icds/management/commands/contact_phone_fix.py

```diff
@@ -0,0 +1,174 @@
+import csv
+import six
+import sys
+import time
+
+from datetime import datetime
+from xml.etree import cElementTree as ElementTree
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.util import SYSTEM_USER_ID
+from corehq.form_processor.backends.sql.dbaccessors import CaseReindexAccessor
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from corehq.util.log import with_progress_bar
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.form_processor.backends.sql.dbaccessors import iter_all_rows
+
+from dimagi.utils.chunked import chunked
+from casexml.apps.case.mock import CaseBlock
+
+DOMAIN = "icds-cas"
+CASE_TYPE = "person"
+
+PHONE_NUMBER_PROPERTY = "contact_phone_number"
+HAS_MOBILE_PROPERTY = "has_mobile"
+HAS_MOBILE_PROPERTY_VALUE = "yes"
+CONTACT_PHONE_NUMBER_IS_VERIFIED = "contact_phone_number_is_verified"
+CONTACT_PHONE_NUMBER_IS_VERIFIED_VALUE = 1
+CASE_ITERATION_COUNT = 10000
+MAX_RESCUE_EXCEPTIONS_ON_UPDATE = 5
+CSV_HEADERS = ['Case Id']
```

Comment:
can you include the original phone number on this?
This would be useful to do a quick analysis of what all we changed later.